src/loader/loader.c

```diff
@@ -195,6 +208,31 @@ load_utility_cmd(Node *utility_stmt)
 static void
 post_analyze_hook(ParseState *pstate, Query *query)
 {
+
+	if (query->commandType == CMD_UTILITY)
```

Comment:
Is it right that this this happens irrespective of guc_disable_load? I think so but want an extra pair of eyes here